components/lane_follower/src/individual_follower.py

```diff
@@ -0,0 +1,172 @@
+# Lane Follower for a single camera
+# Applies a perspective transform to an image, and isolates lane lines
+import numpy as np
+import math
+import cv2
+
+
+class IndividualFollower:
```

Comment:
```suggestion
class LaneFollower:
```

Reply:
Not making this change as lane follower is an overarching class that creates 2 individual followers; one for each camera.